File: aldebaran-orders-micro/aldebaran-order-server/src/main/java/com/aldebaran/server/Startup.java
package com.aldebaran.server;

import com.aldebaran.server.config.SpringConfiguration;
import io.prometheus.client.hotspot.DefaultExports;
import org.springframework.boot.SpringApplication;


public class Startup {

    static {
        DefaultExports.initialize();
    }

    public static void main(String[] args) {
        SpringApplication.run(SpringConfiguration.class, args);
    }
}

File: aldebaran-auth-micro/aldebaran-auth-server/src/main/java/com/aldebaran/auth/server/Startup.kt
package com.aldebaran.auth.server

import com.aldebaran.auth.server.config.SpringConfiguration
import io.prometheus.client.hotspot.DefaultExports
import org.springframework.boot.SpringApplication


fun main(args: Array<String>) {
    DefaultExports.initialize()

    SpringApplication.run(SpringConfiguration::class.java)
}
